kit/azure/bootstrap/main.tf

```diff
@@ -13,7 +12,7 @@ module "terraform_state" {
 
   source                           = "./terraform-state"
   location                         = var.terraform_state_storage.location
-  resources_cloudfoundation        = var.terraform_state_storage.name
+  cloudfoundation_name             = var.terraform_state_storage.name
```

Comment:
d: this reads confusing. Could we name the var just "name" in the `terraform-state` model, then it reads fine